sanityChecks.groovy

```diff
@@ -1,36 +1,29 @@
-def linux_build() {
-    sh 'echo "works"'
-}
-def windows_build() {
-    bat 'echo \'works\''
-}
-timeout(60) {
-    stage('Verify VMAgents templates') {
-        parallel 'Custom Ubuntu without init script': {
-            node('s-cust-no-init-ubuntu') {
-                linux_build()
-            }
-        },
-        /*'Custom Ubuntu with init script': {
-            node('s-cust-init-ubuntu') {
-                linux_build()
-            }
-        },*/
-        'Reference Ubuntu with init script': {
-            node('s-ref-ubuntu') {
-                linux_build()
-            }
-        },
-        'Custom Windows with init script': {
-            node('s-cust-windows') {
-                windows_build()
-            }
-        },
-        'Reference Windows with init script': {
-            node('s-ref-windows') {
-                windows_build()
+#!groovy
+pipeline {
+    agent none
+    options {
+        timeout(time: 60, unit: 'MINUTES')
+    }
+    stages {
+        stage('Verify VMAgents templates') {
+            steps {
+                 parallel ( failFast: true,
+                    'Custom Ubuntu without init script': {
+                        runSanityChecks('s-cust-no-init-ubuntu')
+                    },
+                    /*'Custom Ubuntu with init script': {
```

Comment:
Currently the template for that configuration is broken. My guess is that the custom image vhd is not configured correctly. I'll make some time and investigate it sometime in the future :)
I have updated the comment